core/src/main/scala/tamer/registry/Registry.scala

```diff
@@ -25,13 +27,23 @@ object Registry {
   trait Live extends Registry {
     val client: SchemaRegistryClient
     override final val registry: Service[Any] = new Service[Any] {
-      private[this] final val strategy = new SchemaValidatorBuilder().canReadStrategy().validateLatest()
-      private[this] final def validate[R](toValidate: Schema, writerSchema: Schema): RIO[R, Unit] =
-        Task(strategy.validate(toValidate, List(writerSchema).asJava)).as(())
+      private[this] final val logTask: Task[LogWriter[Task]] = log4sFromName.provide("tamer.Registry.Live")
```

Comment:
this use case makes me think to a possible set of constructors for Zio. Something like
```scala
def log4sFromNam(n: String): Task[LogWriter[Task]] =
    log4sFromName.provide(n)
```
I will try it and see if it's worth it.